### PR TITLE
fixed weather alerts

### LINKED ISSUE #13: Implement weather warnings
A red banner that notifies the user of extreme weather conditions. 

E.g. 
- Typhoons
- Extreme rain
- Snow
- Heat
- Tornado

### PR DESCRIPTION
**Pull request by Chukwuemeka-Mba**
Fixes #13 and #42 

### Added the code that renders the properties of the alert object from the weather API in the react Alert component created by @KhudaDad414 in the **playground**

Screen Shot 2022-01-27 at 1 43 43 AM

Contributors: 
@adizcode @ditsuke

### CODE REVIEW

File: src/components/WeatherInfo/info.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "event": "Heat Advisory",
+      "start": 1599941600,
+      "end": 1536766800,
+      "description": "\n###\n\nTravel may be hazardous due to sudden changes in the weather.\n\nFor road conditions and other traveller information from the Ministry of Transportation, visit https://www.ontario.ca/511, https://www.twitter.com/511Ontario, or call 5-1-1.\n\nPlease continue to monitor alerts and forecasts issued by Environment Canada. To report severe weather, send an email to ONstorm@ec.gc.ca or tweet reports using #ONStorm.\n",
+      "tags": [
+        "Extreme temperature value"
+      ]
+    }
+  ]
+}
```
Comment (KhudaDad414) [inline src/components/WeatherInfo/info.json:26]: Where are we going to parse these data from? I don't see any code for that :thinking:

**Review by KhudaDad414 — CHANGES_REQUESTED** [review]
Added some comments.

File: src/components/WeatherInfo/info.json (hunk 1 above)
Comment (Chukwuemeka-Mba) [reply src/components/WeatherInfo/info.json:26]: The data is to be passed from the openweathermap one call API but it doesn't work yet.

File: src/components/playground/Demo.jsx [hunk 2]
```diff
@@ -102,6 +104,11 @@ function Demo() {
   if (error) {
     return <div>Error: {error.message}</div>;
   }
+
+  const weatherInfo = alertsInfo.alerts.map((item) => (
+    <Alert key="id" item={item} />
```
[the remark this answers is not in the document]
Comment (Chukwuemeka-Mba) [reply src/components/playground/Demo.jsx:109]: I don't think that key would work as it is set currently. We tried that :(

**Comment by gmelodie** [comment]
Opening the netlify deploy shows nothing on my end:
image

File: src/components/WeatherInfo/info.json (hunk 1 above)
Comment (ditsuke) [reply src/components/WeatherInfo/info.json:26]: It'll come in from Demo.js. WIP, making soon a PR but it'll be broken **

**Review by ditsuke — APPROVED** [review]
LGTM. Lets go

**Review by anubhav06 — APPROVED** [review]
LGTM ! Awesome work 🎉

**Review by CIPHERTron — APPROVED** [review]
LGTM 🎉

**Pull request merged**